scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/util/DateTimeComparator.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.autoscaler.scheduler.util;
+
+import java.util.Comparator;
+
+public class DateTimeComparator implements Comparator<SpecificDateScheduleDateTime> {
+
+	public int compare(SpecificDateScheduleDateTime first, SpecificDateScheduleDateTime second) {
+		if ((first == null) && (second == null))
+			return 0;
+		if (first == null)
+			throw new RuntimeException("first object is null in DateTimeComparator");
```

Comment:
That doesn't make it right. Like I said, read the javadocs for the proper exception to throw.